packages/grafana-data/src/types/time.ts

```diff
@@ -86,3 +86,22 @@ export function getDefaultRelativeTimeRange(): RelativeTimeRange {
     to: 0,
   };
 }
+
+/**
+ * Simple helper to quickly create a TimeRange object either from string representations of a dateTime or directly
+ * DateTime objects.
+ * @param from
+ * @param to
```

Comment:
Empty lines. GTM in simply removing them since they should be obvious params.

Reply:
removed